prometheus/exporter/job_exporter.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+import subprocess
+import json
+import sys
+import docker_stats
+import docker_inspect
+import gpu_exporter
+import time
+
+def parseFromLabels(labels): 
+    gpuIds = []
+    labelStr = ""
+
+    for label in labels:
+        if "PAI_CURRENT_GPU_ID" in label:
+            s1 = label.split("=")
+            if len(s1) > 1:
+                s2 = s1[1].replace("\"", "").split(",")
+                for id in s2:
+                    gpuIds.append(id)
+        labelStr += label + ","
```

Comment:
>labelStr += label + "," [](start = 8, length = 23)

add else here? because you need't to add the raw PAI_CURRENT_GPU_ID as metrics label.